cfme/tests/v2v/test_v2v_cancel_migrations.py

```diff
@@ -43,21 +45,26 @@ def cancel_migration_plan(appliance, provider, mapping_data_vm_obj_mini):
         target_provider=provider,
         vm_list=mapping_data_vm_obj_mini.vm_list
     )
+
     if migration_plan.wait_for_state("Started"):
+
+        # Cancell migration after disk migration elapsed a 4 minute time duration
+        migration_plan.in_progress(wait_time=4)
```

Comment:
I am doubtful about calling migration_plan.in_progress when you are waiting for plan to start .

         migration_plan.wait_for_state("Started"): 
Just check once if this is working

Reply:
yes, it should work as wait_for_state("Started") make sure the plan gets approved and moved to in-progress.